modules/optimeraRtdProvider.md

```diff
@@ -0,0 +1,44 @@
+# Overview
+```
+Module Name: Optimera Real Time Date Module
+Module Type: RTD Module
+Maintainer: mcallari@optimera.nyc
+```
+
+# Description
+
+Optimera Real Time Data Module. Provides targeting for ad requests from data collected by the Optimera Measurement script on your site. Please contact [Optimera](http://optimera.nyc/) for information. This is a port of the Optimera Bidder Adapter.
+
```

Comment:
please make extremely prominent disclosure here and in your docs repo pr that external js is sourced and can be configured to be publisher hosted in case they want to lock it to a version, or alternatively, please lock this to a version